### PR TITLE
Updated UI

### PR DESCRIPTION
**Pull request by kaivu1999**
Updated some UI.

### CODE REVIEW

**Comment by ankurshaswat** [comment]
The .gitignore file is for python, not for nodeJS. Fix it

**Comment by ankurshaswat** [comment]
Too many typos in comments.

Console logs not meaningful.

**Comment by ankurshaswat** [comment]
Background repeated instead of stretched.

Chat box overlaps with text when window resized.

Chat image links broken.

File: server.js [hunk 1]
```diff
@@ -237,7 +245,7 @@ io.on('connection', function (socket) {
         }
     });
 
-    socket.on("Cancel Connection", function (username) {
+    socket.on("Cancel Connection", function (username) {// hears for cancel connection may be from any ot the two users
```
Comment (ankurshaswat) [inline server.js]: Typo here
Proper Formatting of comment required

File: server.js [hunk 2]
```diff
@@ -213,16 +213,24 @@ io.on('connection', function (socket) {
...
+        if (user != null) socket.broadcast.to(user).emit("file accepted", data);// 
     });
 
+    socket.on("status" , function (data){
+                console.log("hey");
+                username = data.from;
+               var user = logged_clients[username];
+        socket.broadcast.to(user).emit("status", data);
+    });
+console.log("heyr");
```
Comment (ankurshaswat) [inline server.js]: Proper log messages required
Like function name followed by extra data

File: _gitignore [hunk 3]
```diff
@@ -0,0 +1,101 @@
+# Byte-compiled / optimized / DLL files
```
Comment (ankurshaswat) [inline _gitignore]: Git ignore is not of python

File: index.html [hunk 4]
```diff
@@ -43,7 +69,7 @@ <h3 id="welcomeLine">Welcome!</h3>
                         <form class="navbar-form search-bar" >
                             <div class="form-group">
                                 <input type="text" id="search-input" autocomplete="off" class="form-control" placeholder="Search for users" name="user" onkeyup="dynamicSearch()">
-                                <!--i wonder what would happen if we turn autocomplete on the above tag? Nothing happens we have to place some AJAX there I think-->
+                                <!--i wonder what would happen if we turn autocomplete on the above tag? Nothing happens we have to place somnoe AJAX there I think-->
```
Comment (ankurshaswat) [inline index.html]: Typo

File: index.html [hunk 5]
```diff
@@ -56,8 +82,9 @@ <h3 id="welcomeLine">Welcome!</h3>
                 <!--<br>-->
                 <!-- Search Bar -->
                 <div class="row">
-                    <div class="col-xs-11 col-sm-11 col-md-8" id="user-section">
-                        <h2>Online Users</h2>
+                    <div class="col-xs-11 col-sm-11 col-md-6" id="user-section">
+                        <h2>Online Users
+                        </h2>
```
Comment (ankurshaswat) [inline index.html]: Unnecessary space ??

File: index.html [hunk 6]
```diff
@@ -102,74 +129,145 @@ <h2>User Requests</h2>
...
                     </label>
                     <h4 id="file-desc">No files selected</h4>
                     <br>
-                    <input type="submit" value="Send" id="file-send-button" class="btn btn-default">
+                    <input type="submit" value="Send" id="file-send-button" class="btn btn-success">
                     <!--</form>-->
                     <hr>
                 </div>
+                <div class = "col-lg-4" id="sequence">
+<!--                    <li> Here we will have list of transactions history kinda thing </li>
```
Comment (ankurshaswat) [inline index.html]: Be more detailed

File: index.html [hunk 7]
```diff
@@ -102,74 +129,145 @@ <h2>User Requests</h2>
...
+                                    <strong class="primary-font">Suyash</strong> <small class="pull-right text-muted">
+                                        <span class="glyphicon glyphicon-time"></span>14 mins ago</small>
+                                </div>
+                                <p>
+                                    Dev Club is a community of tech-minded people in IIT Delhi who experience what real world collaborative software development is and allows everyone to pursue big
+                                </p>
+                            </div>
+                        </li>
+                        <li class="right clearfix"><span class="chat-img pull-right">
+                            <img src="/images" alt="User Avatar" class="img-circle" />
```
Comment (ankurshaswat) [inline index.html]: Image link incomplete

File: index.html [hunk 8]
```diff
@@ -102,74 +129,145 @@ <h2>User Requests</h2>
...
-                    <p id="fileBeingSent">  </p>
-                    <div class="progress">
-                        <div class="progress-bar" id="file1" role="progressbar" aria-valuenow="0" aria-valuemin="0" aria-valuemax="100" style="width: 0%;"></div>
-                    </div>
+                        <h3 id="fileProgress">
+                            Progress -
+                        </h3>
+                        <p id="fileBeingSent">  </p>
+                        <div class="progress">
+                            <div class="progress-bar" id="file1" role="progressbar" aria-valuenow="0" aria-valuemin="0" aria-valuemax="100" style="width: 0%;"></div>
```
Comment (ankurshaswat) [inline index.html]: Styles should be in CSS

File: server.js [hunk 9]
```diff
@@ -80,9 +80,9 @@ io.on('connection', function (socket) {
             io.sockets.emit('updateUsersList', Object.keys(connected_clients));
             if (waiting_clients[username] == null) { //if user is not in waiting list
                 // console.log("Test username " + username);
-                socket.broadcast.to(user).emit("offer", {
+                socket.broadcast.to(user).emit("offer", {// may be this code broadcasts only to user who is rquested
```
Comment (ankurshaswat) [inline server.js]: Be sure and then remove comment
No MAYBE(s)

**Comment by ankurshaswat** [comment]
Document the functions if possible
So that we can run an automatic documentation software and store the documentations.

**Comment by ankurshaswat** [comment]
Add feedback and suggestion/bug link
Also user requests extra text should be on next line (These users want to share...)

**Comment by ankurshaswat** [comment]
See the Online users and User requests box.
Their CSS seems different (on resizing different behaviour is observed).

**Comment by kaivu1999** [comment]
Also user requests extra text should be on next line (These users want to share...)
Regarding this I think that would make the two boxes unsymmetrical... Do you still need it to be changed.
I have changed according to above comments

**Pull request merged**